### ISSUE TITLE
Could not locate Gemfile

### ISSUE THREAD

**Issue by AndyObtiva** [alx3dev/executable-ruby#4]
Hi, I just installed the latest gem version and ran `exer --setup`, and here is what I get as an error:

```
% exer --setup

bundle install
+ bundle install
Could not locate Gemfile
```

It seems the gem is currently designed to only work from a cloned executable-ruby directory with the assumption of `Gemfile` existing?

**Comment by alx3dev** [comment]
`exer --setup` should allow it to be installed from `rubygems`, while `bin/setup` for cloned repo, but obviously something is wrong. Thanks for reporting, I will look for solution.

**Comment by alx3dev** [comment]
I closed another issue for name conflict, since both are connected to the same problem - `exer.rb`. I will rewrite it during the day.

**Comment by AndyObtiva** [comment]
By the way, the etiquette is usually to close an issue only once it has been addressed (a release is made or code/fix/advice is fully provided), using the issue as a way of motivation towards doing the work. Otherwise, the software engineer might close the ticket and not get the work done, losing out on the motivation factor. It is important to be patient instead of closing the ticket right away. Patience is good.
